lib/sign_in/view/widgets/login_button.dart

```diff
@@ -13,17 +16,24 @@ class LoginButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final l10n = context.l10n;
+
+    /// TODO: Remove after general app UI validation
+    final User user = User(id: '', isNewUser: false, photo: '', providerId: '', name: '', email: '', phoneNumber: '');
```

Comment:
Please fake a real user with for a example a real link to a profile image, name email etc...